Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/wrapper/impl/ConceptEntryWrapperCreator.java

```diff
@@ -79,6 +91,13 @@
 
                 }
             }
+            
+            //adding reviewflag in wrapper
+            revReq = dbManager.getEntry(entry.getId(),entry.getWordnetId(),true);
+            if(revReq!=null) {
+                wrapper.setReviewFlag(revReq.isReviewFlag());
+                wrapper.setComment(revReq.getComment());
```

Comment:
why not adding the comment object right to the wrapper? no reason to add an extra property for everything we want to know about comments.